lib/selector/tokenizer/tokenizer.go

```diff
@@ -55,7 +55,7 @@ type Token struct {
 
 const (
 	// LabelKeyMatcher is the base regex for a valid label key.
-	LabelKeyMatcher = `([a-zA-Z0-9_./-]{0,253}/)?[a-zA-Z0-9]([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])?`
+	LabelKeyMatcher = `[a-zA-Z0-9_./-]{1,512}`
```

Comment:
The validator (for the API) pulls in this value now which it originally never did - so I think we should be OK.